packages/edit-site/src/components/editor/index.js

```diff
@@ -78,7 +78,7 @@ function Editor() {
 			isFullscreenActive: isFeatureActive( 'fullscreenMode' ),
 			sidebarIsOpened: !! select(
 				interfaceStore
-			).getActiveComplementaryArea( editSiteStore ),
+			).getActiveComplementaryArea( 'core/edit-site' ),
```

Comment:
`editSiteStore.name`?

Reply:
Or how about hacking `getActiveComplementaryArea` to check for objects with `name` property? 😛 So we can avoid these magical strings.

Reply:
https://github.com/WordPress/gutenberg/blob/fd24f069afd5165f33e96d004b2039e94b5f4ceb/packages/edit-post/src/components/sidebar/settings-sidebar/index.js#L43-L45

Reply:
This is how it's done somewhere else in the codebase. I'd still think about making `getActiveComplementaryArea` work with store objects.

Reply:
You can check if it is an object and have `name` inside `getActiveComplementaryArea`. The only question is if that should be based on the store name really. It's hard to tell.

Reply:
IMHO there are no expectations for it to be a store name, so we shouldn't accept a store object as `scope`.
A string is totally fine as it is, and passing `store.name` is perfectly reasonable to me.

One thing I'd point out, though, is that at this point we are (almost everywhere) using the `store.name` for the selector, but we are still creating the `<ComplementaryArea>` components with an hardcoded string as scope.
E.g.
https://github.com/WordPress/gutenberg/blob/68f66ed25c4e0816fbe7ae6cebb8aa970492dc80/packages/edit-post/src/components/sidebar/plugin-sidebar/index.js#L99

It's not a big deal, but we might want to update them all with `store.name` instead, to keep them consistent, or risk breaking them in the off chance a store changes name.